restws_basic_auth/restws_basic_auth.info

```diff
@@ -1,3 +1,5 @@
 name = Basic authentication login
 description = User login from HTTP authorization headers (part of RESTful web services).
+core = 7.x
```

Comment:
Remove `core = 7.x. `

In backdrop this is replaced by 
`backdrop = 1.x`

which you have below. This line will create problems.